Source/Resources/Content/MSGraph/Groups/BUILTIN - Corporate Devices - Physical.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "This group includes all Physical End User Corporate Devices, i.e. those that represent physical machines as opposed to virtual desktops.",
+  "description": "What is it: A dynamic group that includes all physical corporate devices. \n Simeon Notes: This is used to apply policies and applications that are only applicable to physical devices, e.g. BitLocker encryption.",
```

Comment:
**What does this do?**
Creates a group that includes physical corporate devices managed by Intune.

**Why should you use this?**
This group is used to assign applications and policies that should apply to all physical devices (e.g. BitLocker encryption)

**What is the end user impact?**
Users of this group can be assigned applications and policies that should apply only to physical devices. 

**Learn more**
NA